atom/browser/native_window_views.cc

```diff
@@ -107,7 +134,8 @@ NativeWindowViews::NativeWindowViews(
       menu_bar_alt_pressed_(false),
       keyboard_event_handler_(new views::UnhandledKeyboardEventHandler),
       use_content_size_(false),
-      resizable_(true) {
+      resizable_(true),
+      background_color_(SK_ColorWHITE) {
```

Comment:
It seems that we don't need to store this value, it is only used in constructor.